src/nft-card-back.ts

```diff
@@ -181,18 +207,24 @@ export class NftCardBackTemplate extends LitElement {
     `
   }
 
-  @property({type: Object}) public traitData = {}
+  @property({type: Object}) public traitData!: TraitData
   @property({type: Boolean}) public loading = true
-  @property({type: Boolean}) public horizontal
+  @property({type: Boolean}) public horizontal!: boolean
+
+  @property({type: Number}) public cardHeight!: number
+  @property({type: Number}) public cardWidth!: number
+
+  @property({type: Object}) private traits!: Traits
 
-  @property({type: Number}) public cardHeight
-  @property({type: Number}) public cardWidth
+  private static formatTrait(trait: string) {
```

Comment:
small thing but to make this more specific i'd call it `formatTraitType(traitType: string)`